release-notes/WCF-Web-Service-Reference-notes.md

```diff
@@ -4,6 +4,17 @@ WCF Web Service Reference connected services provider is now part of Visual Stud
 
 ## Release History
 
+### 12/7/2017 - Visual Studio 2017 v15.6 Preview 1.0 [27205.0.d15.6]
+
+* Added support of updating an existing service reference. This simplifies the process for regenerating the WCF client proxy code for an updated web service (read [preview 1 blog post](https://blogs.msdn.microsoft.com/visualstudio/2017/12/07/visual-studio-2017-version-15-6-preview/) for more details).
+* Added support for generating code even if the provided service metadata does not contain any endpoints ([#2238](https://github.com/dotnet/wcf/issues/2238)).
+* UI improvements and bug fixes ([#2401](https://github.com/dotnet/wcf/issues/2401))
```

Comment:
#2245 support for resolving WSLD and Schema imports.
#2401 VS hosting service now is started correctly.
#2415 Support for projects targeting net47 and other platforms.
#2197 Fixed problem related to metadata processing